ansible/roles/ansible-horizon-plugin/tasks/pre_tasks.yml

```diff
@@ -2,6 +2,25 @@
 #For Debian: apache2
 #For RHEL: httpd
 
+- name: Find Horizon directory path
+  find:
+    paths: "{{ horizon_virtual_env | dirname }}/"
+    file_type: directory
+    use_regex: yes
+    patterns:
+      - "{{ horizon_virtual_env | basename }}"
+    recurse: no
+  register: horizon_path
+
+- name: Find Horizon directory match count
+  set_fact:
+    horizon_dir_cnt: "{{ horizon_path.matched|int }}"
```

Comment:
the matched dir count value can be checked here directly without setting it into a fact.